src/components/LoginPage.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import * as api from '../utils/api';
+
+function LoginPage({setCurrentUser, currentUser, setIsUserLoggedIn}) {
+
+    const [userInput, setUserInput] = useState('');
+    const [isValidUser, setIsValidUser] = useState(null)
+    const [users, setUsers] = useState([]);
+    
+    const handleValue = (e) => {
+        setUserInput(e.target.value);
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        users.forEach(user => {
+            if (user.username === userInput) {
+                setIsValidUser(true);
+                setCurrentUser(user)
+                setIsUserLoggedIn(true)
+            }
+        })

```

Comment:
It would be good to include some error handling in here to give feedback to a user when they type in a name that does not match a valid user